src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php

```diff
@@ -25,7 +25,7 @@ public function __construct(CachedLabelsApi $labelsApi)
     public function onPullRequest(GitHubEvent $event)
     {
         $data = $event->getData();
-        if ('opened' !== $action = $data['action']) {
+        if (!in_array($action = $data['action'], array('opened', 'edited'), true)) {
```

Comment:
Yeah I thought about that when originally making a subscriber factory. But I really think we should keep the data array (payload) and maybe use some specific events like an `GitHubIssueEvent` which can hold specific method like `getAction` and fallback on the generic event in most case.

This should be the goal of another PR, what do you think?